### ISSUE TITLE
When starting the VM with docker infinite loop of error

### ISSUE THREAD

**Issue by lolaustin** [crybapp/portal#3]
### When starting a VM with the command supplied in portals, an infinite loop of 

in portal:
`Disconnected from @cryb/portals ws
Attempting reconnect to @cryb/portals via WS
Connected to @cryb/portals via WS
ready? true
emitting beacon to portals server
Disconnected from @cryb/portals ws
Attempting reconnect to @cryb/portals via WS
Connected to @cryb/portals via WS
ready? true
emitting beacon to portals server
Disconnected from @cryb/portals ws
Attempting reconnect to @cryb/portals via WS
Connected to @cryb/portals via WS
ready? true
emitting beacon to portals server
Disconnected from @cryb/portals ws
Attempting reconnect to @cryb/portals via WS`

### portals outputs an infinite loop of: 

`Connected to @cryb/portals via WS
ready? true
emitting beacon to portals server
Disconnected from @cryb/portals ws
Attempting reconnect to @cryb/portals via WS
Connected to @cryb/portals via WS`

**Comment by JeDaYoshi** [comment]
Basically, WebSocket between the portal and portals keeps dying.

**Comment by JeDaYoshi** [comment]
Update: After I got installing it myself, I get that loop myself, while I get this in `@cryb/portals`:

```
socket open
recieved message from undefined (unknown) over ws 2 undefined
authentication error PortalNotFound
unknown socket closed
```

I'll be moving this issue to `@cryb/portals`.

**Comment by williamsthing** [comment]
What command are you using to start the `@cryb/portal` image via Docker?

**Comment by JeDaYoshi** [comment]
This should already be fixed along crybapp/portals#8.

**Comment by williamsthing** [comment]
Closing this issue out in favour of https://github.com/crybapp/portals/issues/8 - let me know if there are any more issues @lolaustin.
